rholang/doc/rholangtut-mercury_PL.md

```diff
@@ -0,0 +1,419 @@
+# Instrukcja Rholanga
+
+Rholang to nowy język programowania przeznaczony do stosowania w systemach rozproszonych. Jak wszystkie nowonarodzone rzeczy, szybko się zmienia i rośnie; ten dokument opisuje składnię, która będzie używana w wydaniu Mercury.
+
+Rholang jest "zorientowany na proces": wszystkie obliczenia odbywają się za pomocą przekazywania komunikatów. Wiadomości są przekazywane na "kanałach", które są raczej kolejkami wiadomości, ale zachowują się jak zestawy, a nie kolejki. Rholang jest całkowicie asynchroniczny, w tym sensie, że podczas gdy możesz czytać wiadomość z kanału, a następnie coś z nią zrobić, nie możesz wysłać wiadomości, a następnie zrobić coś po otrzymaniu - przynajmniej nie bez wyraźnego czekanie na wiadomość potwierdzającą z odbiornika.
```

Comment:
* zestawy --> zbiory ?
* przynajmniej nie bez wyraźnego czekani**a**
* wiadomość potwierdzającą od odbiorcy

Reply:
Hi. Thanks for the corrections. I don't know how these mistakes slipped by :) 
I am closing the pull request and opening a new one in 2 days with the updated version of the tutorial (0.3) since this one is outdated.